AdminWebsite/AdminWebsite/Services/ConferenceDetailsService.cs

```diff
@@ -1,62 +1,24 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using AdminWebsite.Configuration;
-using AdminWebsite.Extensions;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Options;
 using VideoApi.Client;
+using VideoApi.Contract.Requests;
 using VideoApi.Contract.Responses;
 
 namespace AdminWebsite.Services
 {
     public interface IConferenceDetailsService
     {
-        Task<ConferenceDetailsResponse> GetConferenceDetailsByHearingIdWithRetry(Guid hearingId, string errorMessage);
-
         Task<ConferenceDetailsResponse> GetConferenceDetailsByHearingId(Guid hearingId, bool includeClosed = false);
     }
     
-    public class ConferenceDetailsService : IConferenceDetailsService
+    public class ConferenceDetailsService(IVideoApiClient videoApiClient) : IConferenceDetailsService
     {
-        private readonly IPollyRetryService _pollyRetryService;
-        private readonly ILogger<ConferenceDetailsService> _logger;
-        private readonly IVideoApiClient _videoApiClient;
-
-        public ConferenceDetailsService(IPollyRetryService pollyRetryService, ILogger<ConferenceDetailsService> logger, IVideoApiClient videoApiClient)
-        {
-            _pollyRetryService = pollyRetryService;
-            _logger = logger;
-            _videoApiClient = videoApiClient;
-        }
-
-        public async Task<ConferenceDetailsResponse> GetConferenceDetailsByHearingIdWithRetry(Guid hearingId, string errorMessage)
```

Comment:
Isn't this implementation used by the front end after a booking to poll for a conference?

Reply:
It used to be when it polled within the admin-web bff, the functionality was moved within the angular client itself and it polls from there instead, and makes individual requests each time. This was left over and isn't used anywhere anymore.